source/components/call-phone.ts

```diff
@@ -29,3 +32,15 @@ export function callPhone(phoneNumber: string, opts?: Options): void {
 		)
 	}
 }
+
+const formatNumber = (phoneNumber: string) => {
+	let re = /(\d{3})-?(\d{3})-?(\d{4})/gu
```

Comment:
Should we handle `+` codes (+1 111 222 3333)?

Reply:
If not here, we should file an issue to handle it in the future.

Reply:
I was also thinking about this. We should probably try to handle that.